.github/actions/report-job-status/action.yml

```diff
@@ -0,0 +1,41 @@
+name: 'Send VK Teams notification on failure'
```

Comment:
Here you use quotes, but you don't use them below in param descriptions. You should choose one option.

Reply:
done